design/FY2022/enhancement_to_simplified_shading.md

```diff
@@ -0,0 +1,74 @@
+
+This post on unmethours.com shows a need for such a simplified shading method in EnergyPlus: https://unmethours.com/question/342/what-assumptions-do-you-make-for-modeling-blinds/ 
+
+## Overview ##
+
+Shading devices in EnergyPlus are modeled as a physical layer inside or outside of a window, or between glass layers within a window. Currently, EnergyPlus models in great detail the convective and radiative interaction between the shading layer and the glass layers and the indoor or outdoor environment, as well as the heat absorbed by the shading device.
+
+The most commonly used *WindowMaterial:Shading* requires nine inputs, which might not be readily available. For example, DOE-2 or eQUEST can model shading devices as a fractional adjustment to reduce the solar transmission from the outside to the inside. When converting such DOE-2 or eQUEST models into an EnergyPlus model, users have to go through a trial-and-error process, as there is no corresponding fractional adjustment in EnergyPlus.
+
+This feature will provide a simplified way of discounting indoor-outdoor solar and visual transmission using a similar fractional adjustment as DOE-2. This allows easier translation between EnergyPlus and other building energy modeling tools. In this feature, the shading will not be modeled as a physical layer as EnergyPlus currently does. It only adjusts the solar and visible transmittance of the corresponding window, without considering the detailed interactions between shading devices and other adjacent objects or environments.
+
+## Approach	##
+
+With the added *WindowMaterial:SimplifiedShade* object, users will be able to input a fractional adjustment α between 0 and 1, either as a constant value or as a schedule. The fractional adjustment will be multiplied to the solar transmittance (`state.dataMaterial->Material(MaterNum).Trans`) and visual transmittance (`state.dataMaterial->Material(MaterNum).TransVis`) of the corresponding windows before entering heat balance calculation. The solar transmittance and visible transmittance of the *WindowMaterial:Glazing* object is from the input fields “Solar Transmittance at Normal Incidence” and “Visible Transmittance at Normal Incidence”. For *WindowMaterial:Glazing:RefractionExtinctionMethod* object, the solar and visible transmittance is derived from user input thickness, solar and visible extinction coefficient, and refraction. For the simple glazing object, the adjustment will be applied to the Solar Heat Gain Coefficient and the Visible Transmittance.
```

Comment:
I have a question about this language: "For the simple glazing object, the adjustment will be applied to the Solar Heat Gain Coefficient and the Visible Transmittance."

I'm not sure that I understand what this means exactly. Can you confirm that these adjustments would _not_ result in the same effect as if someone had adjusted the inputs to the `WindowMaterial:SimpleGlazingSystem` object in the same way? I primarily want to make sure that different [angular properties](https://bigladdersoftware.com/epx/docs/9-6/engineering-reference/window-calculation-module.html#step-7.-determine-angular-performance), derived from U-factor & SHGC, will not be chosen for the window simply based on whether this shading object is used.

I also wonder why this approach is suggested for the simple glazing system. Is it not possible to modify solar/visible transmittance directly, as is done for detailed windows?

Reply:
> My understanding is that if the adjustment factor is 0.5, the SHGC and U-value from user input are both 0.7, then the actual SHGC and U-value in downstream heat balance calculation will be 0.35 and 0.7. It will function the same way as if users input 0.35 and 0.7 as the SHGC and U-value, without configuring the simplified shading.

My point is that I don't think it should behave the same way, because it can result in different detailed window properties being used. I assume the detailed window properties should remain the same whether this shading object is used or not.

> As for the angular properties, I haven't thought of this before. But if we want to maintain the correlation curve, we could keep track of the information and use the correlation curve from the un-adjusted U-value, SHGC combination. With the above example, we'll use curve E instead of curve J in the following diagram. What do you feel about this approach?

Angular properties are just one example. See [the description of the simple window model](https://bigladdersoftware.com/epx/docs/9-6/engineering-reference/window-calculation-module.html#simple-window-model). For example, I assume the layer solar transmittance, layer solar reflectance, and layer visible properties would also need to use the unadjusted values. It feels like trying to use the unadjusted values everywhere they should be used is a risky approach, in that some may be missed. I can't help but wonder if there's a simpler implementation approach (like simply reducing the direct/diffuse solar that the window sees), but I say this without any knowledge of the code. Perhaps @EnergyArchmage or @nealkruis has an opinion here.

Reply:
Maybe we will just adjust the $T_{sol}$ computed as below (https://bigladdersoftware.com/epx/docs/9-6/engineering-reference/window-calculation-module.html#step-4.-determine-layer-solar-transmittance), not adjusting SHGC itself, so that all other calculations use the original un-adjusted values? Any thoughts? @shorowit 

<img width="789" alt="image" src="https://user-images.githubusercontent.com/13421161/165991244-691e1a6d-f695-442b-a185-9f64677e0fc1.png">